pages/tutorials/Creating-a-Terra-World.md

```diff
@@ -0,0 +1,115 @@
+```diff
+! This section is incomplete !
+```
+
+> This guide is intended for the **Bukkit** version of Terra. Guides will be written for other platforms once they
+> mature.
```

Comment:
If you say Bukkit somewhere, it becomes mandatory to say (/Spigot/Paper/Tuinity/etc), since people dont seem to understand how forks work.

Probably would also be good to put that Paper is recommend because we use features provided by their extended API